modules/service_accounts/main.tf

```diff
@@ -132,7 +132,7 @@ resource "google_service_account_iam_member" "workload_binding" {
 ### service account for stackdriver
 resource "google_service_account" "stackdriver" {
   count        = var.enable_stackdriver == true ? 1 : 0
-  account_id   = var.stackdriver_sa_name
+  account_id   = substr("stackdriver-${var.customer_name}-${random_id.main.hex}", 0, 30)
```

Comment:
Could we not add these permissions to the existing sa account?

And if not could we generating the names using the same pattern

```
resource "random_id" "main" {
  # 30 bytes ensures that enough characters are generated to satisfy the service account ID requirements, regardless of
  # the prefix.
  byte_length = 30
  prefix      = "${var.namespace}-sa-"
}
```

Reply:
Yes, we can add permissions to the existing service account. However, there is an existing GitHub issue (https://github.com/wandb/terraform-google-wandb/issues/99) that recommends creating a workload identity in GKE with a service account which have kms and gcs permission.

Reply:
This should be resolved. The random_id has the [var.namespace](https://github.com/wandb/terraform-google-wandb/pull/144/files#diff-33389c5e33c24b5344a73686778f179cf4aab61d2ec481e590b8db73f1363548R8) in it.